software/raft/raft_server_riscv/mica/util/cityhash/city.h

```diff
@@ -0,0 +1,112 @@
+// Copyright (c) 2011 Google, Inc.
```

Comment:
I think this file can be deleted. I originally imported these files from the eRPC repo. They made modifications to `city.{cc,h}` and renamed the files to `city_mod.{cc,h}`. Only one of these files is compiled and linked.

Reply:
I was able to delete city.cc without any issues, but I think the existing code still depends on city.h via city_mod.h, so I'm keeping city.h for now.